04_numeronym/mohitnag/main_test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNumeronyms(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    []string
+		expected []string
+	}{
+		{"Scenario One", []string{"ehi!@  45%!"}, []string{"e9!"}},
+		{"Scenario two", []string{""}, []string{""}},
+	}
+	for _, td := range testData {
+		td := td
+		t.Run(td.Scenario, func(t *testing.T) {
+			numeronym := numeronyms(td.input...)
+			assert.Equal(t, td.expected, numeronym)
+			fmt.Println(numeronym)
+		})
+	}
+}
+func TestMain(t *testing.T) {
+	assert := assert.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := string("[a11y K8s abc]")
+	actual := buf.String()
+	actual = strings.Replace(actual, "\n", ",", -1)
+	actual = strings.TrimRight(actual, ",")
+	assert.Equal(expected, actual)
+}
+
+func TestNumeronym(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    string
+		expected string
+	}{
+		{"Three letters", "abc", "abc"},
+		{"Four letters", "test", "t2t"},
+		{"Test Unicode", "\u00C0\u00C0\u00C0\u00C0", "\u00C02\u00C0"},
```

Comment:
Here again, I think this was mentioned in another pull request, but it is easier to read if you use the actual character, not the unicode value.

Eg. `À` instead of `\u00C0`. Please fix these

Reply:
Done